components/brave_wallet/browser/simple_hash_client_unittest.cc

```diff
@@ -252,31 +252,22 @@ TEST_F(SimpleHashClientUnitTest, GetSimpleHashNftsByWalletUrl) {
 }
 
 TEST_F(SimpleHashClientUnitTest, ParseNFTsFromSimpleHash) {
-  std::string json;
-  std::optional<base::Value> json_value;
-
   // Non dictionary JSON response yields nullopt
-  json = R"([])";
-  json_value = base::JSONReader::Read(json);
-  ASSERT_TRUE(json_value);
-  auto result =
-      simple_hash_client_->ParseNFTsFromSimpleHash(*json_value, true, false);
+  std::string json = R"([])";
+  auto result = simple_hash_client_->ParseNFTsFromSimpleHash(
+      base::test::ParseJsonDict(json), true, false);
   ASSERT_FALSE(result);
```

Comment:
I think we don't need to test non-dict inputs as it is validated by caller now

Reply:
Done. Thanks.